schemes/scheme-model.js

```diff
@@ -0,0 +1,54 @@
+const db = require("../data/db-config");
+
+module.exports = {
+  find,
+  findById,
+  findSteps,
+  add,
+  update,
+  remove
+};
+
+function find() {
+  return db("schemes");
```

Comment:
```suggestion
  return db("schemes")
  .then(schemes => schemes)
```
README asks you to return a promise that will resolve to the array of schemes